mantidproject/mantid
# Scanning workspace construction

A new class, `ScanningWorkspaceBuilder` has been added. This is meant to allow for easier construction of scanning workspaces.

`CreateSampleWorkspace` now supports building a sample scanning workspace using the builder.

A real loader using the `ScanningWorkspaceBuilder` is in progress, currently on the branch `d2b_loading`.

**To test:**

Code review. Check documentation is clear and complete.

Try creating a  scanning workspace with `CreateSampleWorkspace`. Check the dimensions, positions and angles are as expected.

Fixes #19198.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
